docs/aerodromes/Jandakot.md

```diff
@@ -0,0 +1,135 @@
+| **Jandakot ADC**  | **Jandakot Tower**  | **118.100**          | **JT_TWR**                        |
+| **Jandakot SMC**  | **Jandakot Ground**   | **124.300**          | **JT_GND**                       |
+| Jandakot ATIS     |                | 128.650          | YPJT_ATIS                                |
+
+ 
+In the absence of an SMC controller, ADC shall assume the SMC responsibilities.
+<figure markdown>
+![YPJT Maneuvering Area](img/ypjt_maneuvring_area.png){ width="500" }
+  <figcaption>Maneuvering Area</figcaption>
+</figure>
+
+## Airspace
+Tower airspace is defined as between SFC to 1500 FT.
+
+<figure markdown>
+![Jandakot Tower Airspace](img/JTTWR.png){ width="700" }
+  <figcaption>Jandakot Tower Airspace</figcaption>
+</figure>
+
+## Fixed-Wing Operations
+### VFR Inbound Procedures
+VFR aircraft will report inbound at `BOAT`, `POWR`, `OAKF` or `RUSS` at `A015`. JT ACD will instruct aircraft to maintain `A015` or remain outside the control zone (workload permitting). Aircraft will then report again at `ADWD` when inbound from `BOAT` or `POWR`, or `FDL` when inbound from `OAKF` or `RUSS`. They should then be instructed to join the circuit as below:
+
+| VFR Approach Point | RWYs 06  | RWYs 24 | RWY 12 | RWY 30 |
+| ----------------| --------- | ---------- | ---------- | --------- |
+| Adventure World (ADWD)   | *"Join base runway 06L"* | *"Join right downwind runway 24R, maintain A015"*, until the aircraft are clear of RWY 24R departures via `Fiona Stanely Hospital` and `FREM`, then *"Cleared Visual Approach"*  | *"Join final runway 12"* | *"Join downwind ruwnay 30, maintain A015"*, until the aircraft is clear of RWY 30 departures via `YGB`, then *"Cleared Visual Approach"* |
+| Forestdale Lake (FDL)  | *"Join downwind runway 06L, maintain A015"*. Aircraft should fly overhead the airfield between the control tower and the upwind end of the runway and join the circuit. Once established in the circuit, *"Cleared visual approach"* | *Join right downwind runway 24R, maintain A015"*. Aircraft should fly overhead the airfield between the control tower and the upwind end of the runway and join the circuit. Once established in the circuit, *"Cleared visual approach"* | *"Join right downwind runway 12"* | *"Join final runway 30"* |
+
+All aircraft will arrive on runway 06L/24R or 12/30.
+
+!!! note
+    Circuit joining instructions given without an assigned altitude imply clearance to conduct the visual approach. There is no need to clear these aircraft for a visual approach.
+
+### VFR Outbound Procedures
+VFR aircraft should report ready to **JT ADC** with their departure intentions.  A takeoff clearance constitutes a clearance to depart the zone by extending the pilot's requested leg of the circuit.  Aircraft departing the zone into class G airspace will transfer to area frequency upon leaving the zone, **no explicit frequency transfer is given to these aircraft**.
```

Comment:
Maybe mention departures into class C for VFR aircraft is not permitted as per the ERSA

Reply:
Unnecessary for VATSIM I think. Too complex for pilots. Also more of a pilot procedure rather than an ATC procedure.